report_py3o/views/py3o_template.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="py3o_template_configuration_search_view" model="ir.ui.view">
+        <field name="name">py3o.template.configuration.search.view</field>
+        <field name="model">py3o.template</field>
+        <field name="arch" type="xml">
+            <search string="Py3o Templates">
+                <field name="name" />
+                <field name="filetype" />
+                <group string="Group By" name="groupby">
+                    <filter
+                        name="filetype_groupby"
+                        string="File Type"
+                        context="{'group_by': 'filetype'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="py3o_template_configuration_form_view" model="ir.ui.view">
+        <field name="name">py3o.template.configuration.form.view</field>
+        <field name="model">py3o.template</field>
+        <field name="arch" type="xml">
+            <form string="Py3o Templates">
+                <group name="main">
+                    <field name="name" />
+                    <field name="filetype" />
+                    <field name="py3o_template_data" filename="name" />
+                </group>
+            </form>
+        </field>
+    </record>
+    <record id="py3o_template_configuration_tree_view" model="ir.ui.view">
+        <field name="name">py3o.template.configuration.tree.view</field>
+        <field name="model">py3o.template</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="filetype" />
+            </tree>
+        </field>
+    </record>
+    <record id="py3o_template_configuration_action" model="ir.actions.act_window">
+        <field name="name">Py3o Templates</field>
+        <field name="res_model">py3o.template</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+    <menuitem
+        id="py3o_template_configuration_menu"
+        parent="py3o_config_menu"
```

Comment:
I don't see this template submenu in runboat (as user admin):

![image](https://user-images.githubusercontent.com/1033124/168487702-4c91c370-70e9-44bf-a076-b52818d1350b.png)


The same template menu does appear in the 14.0 runboat of this project:

![image](https://user-images.githubusercontent.com/1033124/168487679-7e7d73ff-9570-4552-a123-9b3416e87ea5.png)

Do you know why?

Reply:
I think we should just drop this sub-menu entirely and make *Py3o Templates* be sub-menu of *Reporting*, which seems just fine?

Reply:
Sounds good!

Reply:
This change is mandatory to can see the Py3o models :/